com/metacube/DSA/Assignment4/EmployeeCollection.java

```diff
@@ -0,0 +1,63 @@
+package GET2018.com.metacube.DSA.Assignment4;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Copyright (c) 2018 Metacube.com. All rights reserved.
+ * This class creates collection of employee data structure.
+ * @author Chirag Jain
+ *
+ */
+public class EmployeeCollection {
+
+	private List<Employee> employee = new ArrayList<Employee>();
+	private Set<Integer> employeeId = new HashSet<Integer>();
+	
+	/**
+	 * This function add employees to collection.
+	 * @param empId
+	 * @param name
+	 * @param address
+	 * @return true if employee is added else false.
+	 * @throws CustomException
+	 */
+	public boolean addEmployee(int empId, String name, String address) throws CustomException {
+		boolean flag;
```

Comment:
Don't use this method. It is an overhead. Implement using map as suggested above.